advisories/0010.md

```diff
@@ -0,0 +1,49 @@
+### MCR: runc process.cwd and leaked fds container breakout 
+#### Release Date
+2024-01-31
+#### Overview
+Several container breakouts due to internally leaked fds.
+
+Consult the [upstream advisory][upstream] and [snyk][snyk.io] for details, mitigations, workarounds, and patches.
+#### Affected Products
+Mirantis Container Runtime (MCR) <= 23.0.9
+### Vulnerability Information
+#### CVE Identifier
+[CVE-2024-21626][nvd.nist.gov]
+#### CVSSv3.1
+8.6 (High) CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:C/C:H/I:H/A:H
+#### CWEs
+CWE-403, CWE-668
+#### How to check if your environment is vulnerable
+* `containerd.io` package version is equal or lower (<=) than:
+  - 1.6.28~rc.1-1 for DEB package based OSes
+  - 1.6.28-2.1.rc.1.1 for RPM package based OSes
+#### How to fix
+Upgrade MCR packages to the fixed 23.0.9 version or newer
+* `docker-ee*` packages to:
+  - 23.0.9~4-0 for DEB package based OSes
+  - 23.0.9-4 for RPM package based OSes
+* `containerd.io` package to
+  - 1.6.28~rc.1-2 for DEB package based OSes
+  - 1.6.28-4.1.rc.1.1 for RPM package based OSes
+Please follow the [MCR official installation documentation][mcr.docs]
```

Comment:
Should we mention that a daemon restart is not required?

Reply:
Done